query/query4_test.go

```diff
@@ -988,6 +989,142 @@ func TestCascadeSubQuery1(t *testing.T) {
 	}`, js)
 }
 
+func TestHasEvery(t *testing.T) {
+	query := `{
```

Comment:
add this line as the first line of the test in each new test that you have added. This should fix the upgrade tests. Thanks

```
dgraphtest.ShouldSkipTest(t, "071c85b59fdf80ade88f4571e9cf8c867fae1f6a", dc.GetVersion())
```